### PR TITLE
Update Homepage Function Test & Add SSoid from user

### PR DESCRIPTION
**Pull request by NESOY**
## Changes
1. Update Homepage Function Test ( login, signin )
2. Add SSoid from user ( Email Type )

### CODE REVIEW

**Review by sim4858 — APPROVED** [review]
Checked this request

File: src/main/resources/templates/home.html [hunk 1]
```diff
@@ -15,13 +15,13 @@ <h3 class="masthead-brand">Nelp</h3>
...
-                                <li><a href="/login" methods="GET">로그인</a></li>
-                                <li><a href="/signup" methods="GET">회원가입</a></li>
+                                <li><a id="login" href="/login" methods="GET">로그인</a></li>
+                                <li><a id="signup" href="/signup" methods="GET">회원가입</a></li>
                             </ul>
                             <ul class="nav masthead-nav" th:if="${session.user}!=null">
                                 <li class="active"><a href="#">Home</a></li>
-                                <li><a href="/meetings/new" methods="GET">모임을 만들자</a></li>
-                                <li><a href="/logout" methods="GET">로그아웃</a></li>
+                                <li><a id="new" href="/meetings/new" methods="GET">모임을 만들자</a></li>
```
Comment (cjh5414) [inline src/main/resources/templates/home.html]: please use more specific id naming.

**Review by cjh5414 — CHANGES_REQUESTED** [review]

**Review by cjh5414 — COMMENTED** [review]
In the future, it would be better to divide different functions into other PR.

**Review by NESOY — COMMENTED** [review]
Thank Ur Feedback

**Review by cjh5414 — APPROVED** [review]

**Pull request merged**